lib/noticed/delivery_methods/team.rb

```diff
@@ -0,0 +1,32 @@
+module Noticed
+  module DeliveryMethods
+    class Team < Base
+      def deliver
+        post(url, json: format)
+      end
+
+      private
+
+      def format
+        if (method = options[:format])
+          notification.send(method)
+        else
+          {
+            title: notification.params[:title],
+            text: notification.params[:text],
+            sections: notification.params[:sections],
+            potentialAction: notification.params[:notificationAction]
```

Comment:
Shouldn't the `notificationAction` key be underscored? That way it would be consistent with the rest of the keys in the project
```suggestion
            potentialAction: notification.params[:notification_action]
```

Reply:
Yes, I totally agree. It was just to keep the public API as consistent as possible

Reply:
As long as the other keys don't need changing too much, it should be a good improvement. 👍